### PR TITLE
Update config file for translated content (tr)

### PR DESCRIPTION
**Pull request by ismailarilik**

### CODE REVIEW

**Comment by stomar** [comment]
Thanks for the updated PR, I'll review tomorrow.

File: _config.yml [hunk 1]
```diff
@@ -1399,26 +1399,26 @@ locales:
...
+        user_groups:
+          text: Kullanıcı Grupları
+          url: /tr/community/user-groups/
+          description: |
+            Bölgenizdeki Ruby'cilerle temas kurun.
+        weblogs:
+          text: Web günlükleri
+          url: /tr/community/weblogs/
+          description: |
+            Rubyde neler olduğunu takip edin.
```
Comment (enderahmetyurt) [inline _config.yml]: `'de` olsa mi sanki

File: _config.yml [hunk 2]
```diff
@@ -1399,26 +1399,26 @@ locales:
...
+        ruby_core:
+          text: Ruby Core
+          url: /tr/community/ruby-core/
+          description: |
+            En son Rubynin kaba taraflarının parlatılmasına yardımcı olun.
```
Comment (enderahmetyurt) [inline _config.yml]: `'nin` daha mi guzel olur.

File: _config.yml (hunk 1 above)
Comment (ismailarilik) [reply _config.yml]: 9b389ba

File: _config.yml (hunk 2 above)
Comment (ismailarilik) [reply _config.yml]: 9b389ba

**Comment by stomar** [comment]
Thanks!

**Pull request merged**